invenio_base/app.py

```diff
@@ -216,8 +218,8 @@ def base_app(import_name, instance_path=None, static_folder=None,
     try:
         if not os.path.exists(instance_path):
             os.makedirs(instance_path)
-    except Exception:  # pragma: no cover
-        pass
+    except Exception as ex:  # pragma: no cover
+        app.logger.exception(ex)
```

Comment:
But that function takes at least one argument (Two arguments at least, I am not counting self).